server/db/models/products.js

```diff
@@ -0,0 +1,38 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const Products = db.define('products', {
```

Comment:
model names and definitions should be singular - Products should be Product -  'products' should be 'product'